src/connect/ConnectWizard.ts

```diff
@@ -172,7 +174,7 @@ export class ConnectWizard {
         input: MultiStepInput,
         currentContext: IKubeconfigEnrichedContext,
         resourceType: ResourceType
-    ): Promise<(input: MultiStepInput) => Promise<(input: MultiStepInput) => Promise<(input: MultiStepInput) => Promise<void>>>> {
+    ) {
```

Comment:
Has the return type changed?

Reply:
Yes, added an extra step to select the container's. so the return type has changed.